tools/format/swf/instance/MovieClip.hx

```diff
@@ -47,7 +47,7 @@ class MovieClip extends flash.display.MovieClip {
 	@:noCompletion private var activeObjects:Array<ChildObject>;
 	
 	#if flash
-	@:noCompletion private var __currentFrame:Int;
+	@:noCompletion private var __currentFrameOneIndexed:Int;
```

Comment:
👷 🚧 **WARNING**: I do not feel I understand the tools version of the `MovieClip` class, and while these changes seem to satisfy the build, you should really check them to be sure the original intent is still functional. With or without them, I couldn't tell the difference. 

It would help to get an explanation from you @jgranick here.